jobs/branchDay.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const calendar_config = require('../creds').calendar;
+const calendar_API = require('node-google-calendar');
+const moment = require('moment');
+const build_calendar = new calendar_API(calendar_config);
+
```

Comment:
if you're planning to use this again you should break whatever you need out and put it in the dependencies